Entity/CustomFieldOption.php

```diff
@@ -164,31 +163,31 @@ public function setOrder(?int $order): void
     /**
      * {@inheritdoc}
      */
-    public function offsetExists($offset)
+    public function offsetExists(mixed $offset): bool
     {
         return isset($this->{$offset});
     }
 
     /**
      * {@inheritdoc}
      */
-    public function offsetGet($offset)
+    public function offsetGet(mixed $offset): bool
     {
         return $this->offsetExists($offset) ? $this->{$offset} : null;
     }
```

Comment:
The return type declaration breaks the function. e.g. `$this->label` returns a `string`
```suggestion
    #[ReturnTypeWillChange]
    public function offsetGet(mixed $offset)
    {
        return $this->offsetExists($offset) ? $this->{$offset} : null;
    }
```
or explicitly
```suggestion
    public function offsetGet(mixed $offset): CustomField|string|int|null
    {
        return $this->offsetExists($offset) ? $this->{$offset} : null;
    }
```